src/BuiltInTools/dotnet-format/dotnet-format.csproj

```diff
@@ -23,8 +23,7 @@
       These identifiers are for generating the shim'd core executables for signing. Not all options
       from $(RoslynPortableRuntimeIdentifiers) work or make sense in this context.
     -->
-    <PackAsToolShimRuntimeIdentifiers Condition="'$(DotNetBuild)' != 'true'">win-x64;win-x86;osx-x64</PackAsToolShimRuntimeIdentifiers>
-    <PackAsToolShimRuntimeIdentifiers Condition="'$(DotNetBuild)' == 'true' and $(SignableShimRuntimeIdentifier.Contains('$(TargetRid)'))">$(TargetRid)</PackAsToolShimRuntimeIdentifiers>
+    <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64</PackAsToolShimRuntimeIdentifiers>
```

Comment:
What about source-build?

Reply:
I see this in the history:

![image](https://github.com/user-attachments/assets/e2fd8350-7376-46d0-808c-dd7d49febcad)

Reply:
Perhaps I should add the same condition that we use in `aspnetcore` - https://github.com/dotnet/aspnetcore/pull/61282

`'$(DotNetBuild)' != 'true' OR '$(TargetOsName)' == 'win'`

Reply:
I think DotNetBuildSourceOnly not true is the right condition here.

Reply:
OK - will fix it.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/48153/commits/25ab7de24708502ddb7e9d62bcce86248b179d8f